.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Description
+
+Please provide a brief description of the changes in this pull request.
+
+### Motivation
+
+Explain the motivation behind these changes and why they are necessary.
```

Comment:
I think we can replace description and motivation section with just asking people to create an issue first and provide those there. So, it basically will be: ensure that there's an issue with this information

Reply:
Done.